### DISCUSSION TITLE
Userscripts

### DISCUSSION THREAD

**Discussion by Gottarocket** [bromite/bromite#1521] [Ideas]
Would love to have dedicated resource on bromite official site or here with all user scripts that run on bromite specifically as most listed on the resource website either doesnt work on bromite or android

**Comment by uazo** [comment]
> Would love to have dedicated resource on bromite official site

I don't think that will ever happen, it would mean checking every script every time it is released and the purpose is beyond bromite's goal.
However, as a creator, I would love to have feedback on usage.

> all user scripts that run on bromite

an example "Stop link redirection in Google" from https://github.com/Rob--W/dont-track-me-google also listed in https://chromium.woolyss.com/ it works, although not always, but I don't think it's a bromite problem

**Reply by grant** [reply]
You mentioned me, how can I remove myself from these scripts?
